Dockerfile

```diff
@@ -41,8 +42,9 @@ RUN mkdir -p /usr/src/app \
 USER chrome
 WORKDIR /usr/src/app
 
-ENV CHROME_BIN=/usr/bin/chromium-browser \
+ENV CHROME_BIN=/usr/bin/chromium-browser                                            \
+    CHROMIUM_FLAGS=--headless --disable-dev-shm-usage --disable-software-rasterizer \
     CHROME_PATH=/usr/lib/chromium/
 
 # Autorun chrome headless
-ENTRYPOINT ["chromium-browser", "--headless", "--use-gl=swiftshader", "--disable-software-rasterizer", "--disable-dev-shm-usage"]
+ENTRYPOINT ["chromium-browser"]
```

Comment:
Looks like `--use-gl=swiftshader` was dropped in the process.

Reply:
I don't find any doc about that, do you have any pointer on this?

Reply:
@zigarn without putting this in you get "WebGL fallback, SwANGLE (ANGLE + SwiftShader Vulkan)",

`--use-gl=angle --use-angle=swiftshader-webgl`

With this you get the "legacy SwiftShader Open GL ES libraries"

Not exactly sure of the difference, but it's swiftshader either way. You can see it in the code here https://source.chromium.org/chromium/chromium/src/+/main:headless/public/headless_browser.cc;drc=75425ebb5303c7bb416b2261e9b27114ec903d3f;l=40